packages/ses-ava/src/ses-ava-test.js

```diff
@@ -75,13 +75,29 @@ const overrideList = [
   'only',
 ];
 
+/**
+ * @param {Logger} tlog
+ * @param {Logger | 'tlog'} loggerOrOption
+ * @returns {Logger}
+ */
+const getLogger = (tlog, loggerOrOption) => {
+  if (typeof loggerOrOption === 'function') {
+    return loggerOrOption;
+  } else if (loggerOrOption === 'tlog') {
+    // TODO wrap with SES causal console
+    return tlog;
+  } else {
+    throw Error(`Unrecognized loggerOrOption ${loggerOrOption}`);
+  }
+};
+
 /**
  * @template {import('ava').TestFn} T
  * @param {T} testerFunc
- * @param {Logger} logger
+ * @param {Logger | 'tlog'} loggerOrOption
  * @returns {T} Not yet frozen!
  */
-const augmentLogging = (testerFunc, logger) => {
+const augmentLogging = (testerFunc, loggerOrOption) => {
   const testerFuncName = `ava ${testerFunc.name || 'test'}`;
```

Comment:
I'd probably deal with the logger argument here, to validate early and simplify `wrapImplFunc` to `return logErrorFirst(fn, [t, ...args], source, logger || t.log)`.
```suggestion
  const testerFuncName = `ava ${testerFunc.name || 'test'}`;
  let logger;
  if (typeof loggerOrOption === 'function') {
    logger = loggerOrOption;
  } else if (loggerOrOption !== 'tlog') {
    throw Error(`Unrecognized loggerOrOption ${loggerOrOption}`);
  }
```

Reply:
@gibson042 , now that I've got the case where the causal console wraps on `t.log`, I no longer see the opportunity for this simplification. Please PTAL and let me know if I missed it. Thanks.

Reply:
Successor #2109 is no longer options driven. It provides one built in behavior. Thus, on behalf of #2109 , I'm resolving this conversation.